pkg/data/management/podadmissionconfigurationtemplate_data.go

```diff
@@ -21,3 +25,51 @@ func addDefaultPodSecurityAdmissionConfigurationTemplates(management *config.Man
 	}
 	return nil
 }
+
+func newPodSecurityAdmissionConfigurationTemplateRestricted() *v3.PodSecurityAdmissionConfigurationTemplate {
+	return &v3.PodSecurityAdmissionConfigurationTemplate{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "restricted",
+		},
+		Description: "The default restricted pod security admission configuration template",
+		Configuration: v3.PodSecurityAdmissionConfigurationTemplateSpec{
+			Defaults: v3.PodSecurityAdmissionConfigurationTemplateDefaults{
+				Enforce:        "restricted",
+				EnforceVersion: "latest",
+				Audit:          "restricted",
+				AuditVersion:   "latest",
+				Warn:           "restricted",
+				WarnVersion:    "latest",
+			},
+			Exemptions: v3.PodSecurityAdmissionConfigurationTemplateExemptions{
+				Usernames:      []string{},
+				RuntimeClasses: []string{},
+				Namespaces:     strings.Split(settings.SystemNamespaces.Get(), ","),
```

Comment:
I'm not sure this is the best way to do this.... there are a LOT of system namespaces in there that don't necessarily need PSA exemptions.

Reply:
I know it is an overkill, but other opinions are unnecessarily complex:
option 1: hardcode and maintain the list inside the object instead of getting it from settings.SystemNamespaces
option 2:  an extra controller or mutator to add the PSP labels to the namespaces used by rancher and rancher-managed APPs.

Reply:
It's not just a matter of it being overkill, this is borderline a security risk if you ask me. I would be much happier if we had an additional setting for specific PSA-exempt namespaces. This would allow much more granularity. Additionally, we should probably have a cluster-level PSA exemption for namespaces, if the data structure for that won't be too complicated.

Reply:
Based on the offline discussion, we will list the namespaces used by feature charts in the PSACT instead of getting them from the settings.SystemNamepaces.